hcloud/cloud.go

```diff
@@ -61,107 +44,96 @@ type cloud struct {
 	client       *hcloud.Client
 	instances    *instances
 	loadBalancer *loadBalancers
+	config       config.HCCMConfiguration
```

Comment:
Does it make sens to use a global here ?

Reply:
Perhaps, I will do some more refactoring on the usage of `HCCMConfiguration` in `hcops` and the other packages in a follow up PR and will reevaluate then.